test/generate_property_json.js

```diff
@@ -122,19 +145,19 @@ function addReference () {
   return { op: 'reference', path: p }
 }
 
-function stopOrRestart () {
+function stopOrRestartWatcher () {
   if (running) {
     running = false
-    return { op: 'stop' }
+    return { op: 'stop_watcher' }
   } else {
     running = true
-    return { op: 'restart' }
+    return { op: 'restart_watcher' }
   }
 }
 
 function sleep () {
-  const s = 2 ** (1 + faker.random.number(3))
-  return { op: 'sleep', duration: 1000 * s }
+  const s = 2 ** (6 + faker.random.number(8))
```

Comment:
Magic number, hard to maintain and understand why, what and how.

Reply:
We are in a script that generates some random scenario for property based testing. There are no right or wrong values here: if you change the number, it will change what is tester. With short sleeps, race conditions are more tested, with long sleeps, it is more the general synchronization workflow.